src/GridInterpolations.jl

```diff
@@ -108,11 +108,11 @@ function Base.show(io::IO, grid::AbstractGrid)
     end
 end
 
-function ind2x(grid::AbstractGrid, ind::Int)
+function ind2x(grid::AbstractGrid{D}, ind::Int) where D
     ndims = dimensions(grid)
-    x = Array{Float64}(undef, ndims)
+    x::MVector{D, Float64} = @MVector zeros(D)
```

Comment:
if we want to reduce allocation to zero, adding a container for `x` in grid might do it. @zsunberg @mattuntergassmair

Reply:
You mean adding a `MVector{D, Float64}` as a field to the `RectangleGrid` struct, so we can simply use that to store the result instead of allocating - did I understand the suggestion correctly?
It would help with the allocations, but I think what would happen is that if you call `ind2x()` twice, and you are still holding on to the result computed in the first call, then it gets overwritten in the second call without the user noticing, right?

Reply:
I guess in that case users should just pre-allocate memory on their side and use `ind2x!()` rather than `ind2x()`

Reply:
x is typically pretty low-dimensional, right? I think ind2x should return an SVector. I think that should get rid of all the allocations and save a large amount of time.

In fact, I think the calls between ind2x and ind2x! should be switched, i.e. ind2x should contain the logic, and ind2x! should be implemented as
```julia
ind2x!(grid, ind, x)
x[:] = ind2x(grid, ind)
end
```